vclib/src/commonMain/resources/grammar/JsonPath.abnf

```diff
@@ -0,0 +1,149 @@
+; specification: https://datatracker.ietf.org/doc/rfc9535/
```

Comment:
isn't this file part of the JSONPath library now? Same for the `g4` ones

Reply:
fixed in 3e4c8cd54334e7b7ed0bb56817ae64d8ed0d8d0d